jpa/hibernate6/src/main/java/org/jboss/as/jpa/hibernate/JpaLogger.java

```diff
@@ -46,6 +47,10 @@ public interface JpaLogger extends BasicLogger {
     @Message(id = 20261, value = "Hibernate ORM did not register LifeCycleListener")
     IllegalStateException HibernateORMDidNotRegisterLifeCycleListener();
 
+    @LogMessage(level = WARN)
+    @Message(id = 20262, value = "Application custom cache region setting is ignored %s=%s")
+    void ignoredCacheRegionSetting(String propertyName, String setting );
```

Comment:
maybe the message could suggest which property is expected to be used instead in Wildfly?